examples/signals/src/lib.rs

```diff
@@ -19,8 +19,16 @@ impl SignalEmitter {
 
         builder.add_signal(init::Signal {
             name: "something_happens_with_data",
-            // optionally user can specify predefined list of signal arguments using init::SignalArgument
-            args: &[],
+            // this predefined list of signal arguments is optional.
+            // Editor needs it to generate correct argument lists from UI.
+            // Signal will work as good as previous even with empty args
+            args: &[init::SignalArgument {
+                name: "data",
+                default: Variant::from_i64(100),
+                // some insight about hint and usage can be found in the "spinning_cube" tutorial
```

Comment:
It is true that there are a few more examples in that example, but I don't think it's worth noting here: they're neither very special, nor explained in-depth.